src/utils/positionUtils.ts

```diff
@@ -256,13 +257,24 @@ export const calcTradeFee = ({
   marketSnapshot,
   isMaker,
   direction,
+  positionFee = 0n,
+  referralFee = 0n,
 }: {
   positionDelta: bigint
   marketSnapshot?: MarketSnapshot
   isMaker: boolean
   direction: PositionSide
+  positionFee?: bigint
```

Comment:
this should come from the market Parameter in the snapshot rather than being passed in